src/Microsoft.ML.Data/Data/SchemaDefinition.cs

```diff
@@ -68,30 +68,95 @@ public VectorTypeAttribute(params int[] dims)
     /// column encapsulates.
     /// </summary>
     [AttributeUsage(AttributeTargets.Field | AttributeTargets.Property, AllowMultiple = false, Inherited = true)]
-    public sealed class ColumnAttribute : Attribute
+    public sealed class LoadColumnAttribute : Attribute
     {
-        public ColumnAttribute(string ordinal, string name = null)
+        /// <summary>
+        /// Initializes new instance of <see cref="LoadColumnAttribute"/>.
+        /// </summary>
+        /// <param name="ordinal">The index of the column in the text file.</param>
+        /// <param name="name">The optional name of the column, if it should be different from the name of the field or property where the attribute is positioned.</param>
+        /// <param name="loadAllOthers">Wheather to load all columns, besides the one specified in the ordinal.</param>
+        public LoadColumnAttribute(int ordinal, string name = null, bool loadAllOthers = false)
+             : this(ordinal.ToString(), name)
+        {
+            Sources = new List<TextLoader.Range>();
+            var range = new TextLoader.Range(ordinal);
+            range.AllOther = loadAllOthers;
+            Sources.Add(range);
+        }
+
+        /// <summary>
+        /// Initializes new instance of <see cref="LoadColumnAttribute"/>.
+        /// </summary>
+        /// <param name="start">The starting column index, for the range.</param>
+        /// <param name="end">The ending column index, for the range.</param>
+        /// <param name="name">The optional name of the column, if it should be different from the name of the field or property where the attribute is positioned.</param>
+        /// <param name="columnIndexes">Distinct text file column indces to load as part of this column.</param>
+        public LoadColumnAttribute(string start, string end = null, string name = null, int[] columnIndexes = null)
```

Comment:
>columnIndexes [](start = 94, length = 13)

I think this parameter is more misleading than it is helpful. Can you remove it?